app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -0,0 +1,79 @@
+---
+title: If you have a degree
+related_content:
+    Get support training to teach if you're disabled: "/get-support-training-to-teach-if-you-are-disabled"
+    Improve your subject knowledge : "/improve-your-subject-knowledge"
+    Tips on applying for teacher training : "/tips-on-applying-for-teacher-training"
+extra_content:
+    - content/train-to-be-a-teacher/if-you-have-a-degree/events-near-you
+---
+
+If you have a degree, or you’re currently studying for one, there are different routes you can take to getting [qualified teacher status](/what-is-qts) (QTS). You need QTS to work in the majority of schools in England including state maintained primary, secondary and special schools.
+
+We also have information for people outside of the UK who want to get [international qualified teacher status](/international-qualified-teacher-status) (iQTS) or [come to teach in England](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk).
+
+##Study as a postgraduate student
+You can study for [a PGCE or PGDE](/what-is-a-pgce) (an academic qualification) at a university. This is one way to get QTS. Most of the time you will be in schools on placement but there is some academic learning. Courses typically take one year but you may be able to study part-time.
```

Comment:
On Find, they talk about 'PGCE with QTS' - I think we should be consistent rather than say just study for a PGCE?